packages/aws-cdk-lib/aws-cloudtrail/lib/cloudtrail.ts

```diff
@@ -262,6 +270,22 @@ export class Trail extends Resource {
       },
     }));
 
+    if (props.isOrganizationTrail) {
```

Comment:
Could also add a warning in the case that `isOrganizationTrai: true` and `orgId` is not provided. Probably not an error even though it would fail to deploy since some users may have manually added the missing policy already and won't need to use the new `orgId` prop. 

Warning could be something like this but any alternative suggestions are welcome:
`Organization Trails require passing in an organization id using the orgId property to attach a missing policy.`

Reply:
Oh ok that makes a lot more sense!